client/src/admin/FeatureLayer.js

```diff
@@ -0,0 +1,65 @@
+import React, { PropTypes } from 'react';
+import 'leaflet';
+import esri from 'esri-leaflet';
+
+const UNSELECTED_STYLE = {
```

Comment:
I don't think that this is CSS, weight is not a css property, this is styling for the layers and I think it's custom to leaflet